### PR TITLE
Create ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafkk

Ticket: [DM-3901](https://cumulocity.atlassian.net/browse/DM-3901)
Original PR: [7162](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7162)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: After deletion in data grid user stays on current page.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md]: The Claude AI suggested to use the following title:
```suggestion
title: Data grid deletion no longer redirects to first page
```

File: content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3901
+version: 1020.28.4
+---
+After deletion in data grid user stays on current page.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, when deleting an entry in a data grid that was not on the first page, users were unexpectedly redirected back to the first page of the grid. This behavior has now been changed. After deleting an entry, users will remain on the same page they were on before the deletion. This change affects all data grids throughout the application where deletion is possible.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data grid deletion no longer redirects to first page
```
Comment (MWindel) [inline content/change-logs/device-management/ui-c8y-1020-28-4-after-deletion-in-data-grid-user-stays-on-current-page.md]:
```suggestion
title: Data grid deletion no longer redirects to the first page
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**